recipes/petsird-python/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "petsird" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/petsird-{{ version }}.tar.gz
+  sha256: a5d346b1e0a594fa1ab3c5a630e7f24e3fc33a60bcce801aeb06f5648c997b68
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=42
+    - setuptools-scm >=3.4
+    - pip
+  run:
+    - python >={{ python_min }}
+    - numpy >=1.22
+
+test:
+  imports:
+    - petsird
+  commands:
+    - pip check
+  requires:
+    - python {{ python_min }}
+    - pip
+
+about:
+  summary: Library and tools for working with PET Emission Tomography Standardization Initiative Raw Data (PETSIRD) data
```

Comment:
```suggestion
  summary: Library and tools for working with Positron Emission Tomography Standardization Initiative Raw Data (PETSIRD) data
```

Reply:
Happy to change this, but this is auto generated from whatever there is on pypi ;)